react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class ValidatedInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { password: '' };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.validate = this.validate.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ password: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+  }
+
+  validate() {
+    let passwordLength = this.state.password.length;
```

Comment:
`passwordLength` should be a `const` since it wont change during the run of the method